string_width/README.md

```diff
@@ -0,0 +1,75 @@
+# string_width
```

Comment:
We don't create README in deno_std anymore. Instead we write module-level documentation in `mod.ts` with `@module` jsdoc tag.